src/main/java/org/folio/services/ftp/SftpUploadService.java

```diff
@@ -25,7 +26,7 @@
 import java.util.concurrent.CompletionException;
 import java.util.concurrent.TimeUnit;
 
-public class SftpUploadService {
+public class SftpUploadService implements FileExchangeConnectionInfo {
```

Comment:
I think we can rename interface to FileUploadService

Reply:
But we don't describe methods related with file uploading in the interface. We just test connection

Reply:
It would by easier in future to migrate remaining methods to interface, but I am okay to leave as is, just need to rename 'info' to something else, because info commonly used for naming dtos

Reply:
in general very good refactoring, thanks

Reply:
Renamed as you said. Thank for the review